src/mol-script/transpilers/pymol/keywords.ts

```diff
@@ -0,0 +1,272 @@
+                            B.core.set.has([
+                                B.core.type.set(ResDict.nucleic.concat(ResDict.protein).concat(ResDict.solvent)),
+                                B.ammp('label_comp_id')
+                            ])
+                        ]),
+                        'group-by': B.ammp('residueKey')
+                    }),
+                    test: B.core.logic.not([
+                        B.core.set.has([
+                            B.struct.atomSet.propertySet([B.acp('elementSymbol')]),
+                            B.es('C')
+                        ])
+                    ])
+                })
+            ]),
+            property: B.ammp('residueKey')
+        }))
+    },
+    solvent: {
+        '@desc': 'All water molecules. The hardcoded solvent residue identifiers are currently: HOH, WAT, H20, TIP, SOL.',
+        abbr: ['sol.'],
+        map: () => B.struct.generator.atomGroups({
+            'residue-test': B.core.set.has([
+                B.core.type.set(ResDict.solvent),
+                B.ammp('label_comp_id')
+            ])
+        })
+    },
+    guide: {
+        '@desc': 'All protein CA and nucleic acid C4*/C4',
+        map: () => B.struct.combinator.merge([
+            B.struct.generator.atomGroups({
+                'atom-test': B.core.rel.eq([
+                    B.atomName('CA'),
+                    B.ammp('label_atom_id')
+                ]),
+                'residue-test': B.core.set.has([
+                    B.core.type.set(ResDict.protein),
+                    B.ammp('label_comp_id')
+                ])
+            }),
+            B.struct.generator.atomGroups({
+                'atom-test': B.core.set.has([
+                    h.atomNameSet(['C4*', 'C4']),
+                    B.ammp('label_atom_id')
+                ]),
+                'residue-test': B.core.set.has([
+                    B.core.type.set(ResDict.nucleic),
+                    B.ammp('label_comp_id')
+                ])
+            })
+        ]),
+    },
+    metals: {
+        '@desc': 'All metal atoms (new in PyMOL 1.6.1)'
+    },
+    backbone: {
+        '@desc': 'the C, N, CA, and O atoms of a protein and the equivalent atoms in a nucleic acid.',
+        abbr: ['bb.'],
+        map: () => backboneExpr()
+    },
+    'bFCLHMz55tjm16c9': {
+        '@desc': 'protein',
+        abbr: ['polymer.protein'],
+        map: () => B.struct.generator.atomGroups({
+            'residue-test': B.core.set.has([
+                B.core.type.set(ResDict.protein),
+                B.ammp('label_comp_id')
+            ])
+        })
+    },
+    'bFCLHMz55tjm16c0': {
+        '@desc': 'nucleic acid',
+        abbr: ['polymer.nucleic'],
+        map: () => B.struct.generator.atomGroups({
+            'residue-test': B.core.set.has([
+                B.core.type.set(ResDict.nucleic),
+                B.ammp('label_comp_id')
+            ])
+        })
+    }
```

Comment:
Where do these come from?

Reply:
I intended to mimic polymer.protein and polymer.nucleic by these keywords. 

These keywords are found in PyMOL 2.1 or later and I heavily use when selecting protein and nucleic acid seperately. 
https://pymolwiki.org/index.php/Selection_Algebra

I could not use dots in the key of this keyword dict, so I defined them as random string and make abbr. corresponds to actual keywords. Is there any ways to work this around?

The definition is taken from VMD and modified to fit in PyMOL residue dictionary, but these definition are not strict because they also select  ligands (not-polymer) amino-acid residues such as in amino-acyl-tRNA (like 1ttt). 

There's not problem to remove these keywords if you worry about the definitions are not strict to PyMOL's

Reply:
OK, thanks. You can actually use the dots in the keyword. At least it works for me, pushed a change. I think there are close enough for now. It is the same as for the polymer keyword I added initially. One thing I always try it to use the documentation from the PyMOL, Jmol, VMD docs in the transpiler. I changed that too.

Reply:
Thank you for modifying the key and adding explanation for PyMOL version.